torch/csrc/jit/codegen/cuda/scheduler/utils.cpp

```diff
@@ -2316,6 +2454,281 @@ size_t expandVectorizationToContigMergedDomains(
   return word_size;
 }
 
+TORCH_CUDA_CU_API DisjointSets<IterDomain*> disjointViewSets(Fusion* fusion) {
```

Comment:
nit: We can omit the `TORCH_CUDA_CU_API` in a cpp file
```suggestion
DisjointSets<IterDomain*> disjointViewSets(Fusion* fusion) {
```